lib/views/admin/sparql.js

```diff
@@ -100,6 +100,8 @@ function post (req, res) {
 
   const queryString = generator.stringify(query)
 
+  console.log(queryString)
```

Comment:
Could this be console.debug or other more verbose level?

Reply:
Done